appoetry1/Flows/MainFlow.swift

```diff
@@ -24,11 +24,6 @@ class MainFlow: FlowController {
         guard let vc = mainViewController else {
             fatalError("Could not get main vc")
         }
-        
-        //        vc.showMeAgain = { [weak self] in
-        //            self?.pushAgain()
-        //        }
-        
-        self.rootController.setViewControllers([vc], animated: false)
+        self.rootController.pushViewController(vc, animated: true)
```

Comment:
Does the main flow start in a specific way? Or you're just presenting the 'rootController' with attached vc? 
The question is - is there a necessity to perform a 'push' operation here? If not, you can leave 'setViewControllers' in 'start()' function and have the animation set to 'false', since its not doing anything visible anyway